src/TMap.cpp

```diff
@@ -625,7 +625,7 @@ void TMap::initGraph()
         l.id = itRoom.key();
         // locations is std::vector<location> and (locations.at(k)).id will give room ID value
         locations.push_back(l);
-        // Map's usable TRooms (key) to index of entry in locations (for route finding), will lose invalid and unusable (through locking) rooms
```

Comment:
Not sure I grasp the correct meaning, but this seemed ok before

Reply:
"Maps" is the plural singular of "to map" here, so the apostrophe is wrong

Reply:
I.e. a complete sentence would be "This statement maps usable keys to index of …".

Reply:
You're right!

```suggestion
             // This statement maps usable TRooms keys to index of entry in locations (for route finding), will lose invalid rooms and rooms which are unusable through locking
```

Let's write it understandably then.